Forward port release notes for v0.18.2 (#5381)
scikit-image/scikit-image: doc/release/release_0.18.rst

scikit-image 0.18.2
===================

We're happy to announce the release of scikit-image v0.18.2!

scikit-image is an image processing toolbox for SciPy that includes algorithms
for segmentation, geometric transformations, color space manipulation,
analysis, filtering, morphology, feature detection, and more.

This release mostly serves to add wheels for the aarch64 architecture, as well

> as fixing a couple of minor bugs.

Reviewer: "(...) serves to add (...) as well as (to) fix (...)" although it doesn't really *serve* to fix, rather it *happens* to fix minor bugs as well... :thinking: 
Maybe: "This release mostly serves to add wheels for the aarch64 architecture; it also
fixes a couple of minor bugs."

Author: Can you make that a suggestion so I can click and commit? :smile:

Reviewer: Sure! Approving with the suggested change.

> This release mostly serves to add wheels for the aarch64 architecture, as well
> as fixing a couple of minor bugs.

Reviewer: ```suggestion
This release mostly serves to add wheels for the aarch64 architecture; it also
fixes a couple of minor bugs.
```

For more information, examples, and documentation, please visit our website:

https://scikit-image.org

Bug fixes
---------
- allow either SyntaxError or OSError for truncated JPG (#5315, #5334)
- Fix sphinx: role already being registered (#5319, #5335)

Development process
-------------------
- Update pyproject.toml to ensure pypy compatibility and aarch compatibility (#5326, #5328)
- Build aarch64 wheels (#5197, #5210)
- See if latest Ubuntu image fixes QEMU CPU detection issue (#5227, #5233)
- Rename `master` to `main` throughout (#5243, #5295)
- Fixup test for INSTALL_FROM_SDIST (#5283, #5296)
- Remove unecessary manual installation of packages from before_install (#5298)
- Use manylinux2010 for python 3.9+ (#5303, #5310)
- add numpy version specification on aarch for cpython 3.8 (#5374, #5375)

7 authors added to this release [alphabetical by first name or login]
---------------------------------------------------------------------
- François Boulogne
- Janakarajan Natarajan
- Juan Nunez-Iglesias
- John Lee
- Mark Harfouche

> - MeeseeksMachine

Reviewer: Should this bot be included as an author?

Author: I like to give it credit. :smiley: We should keep the robots happy to postpone the robot uprising! :joy:

Reviewer: Ok, ok!!
